client/dashboard/me/tax-details/index.tsx

```diff
@@ -1,11 +1,167 @@
+import { useResetSupportInteraction } from '@automattic/help-center/src/hooks/use-reset-support-interaction';
+import {
+	Card,
+	CardBody,
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+} from '@wordpress/components';
+import { useDispatch as useDataStoreDispatch, useDispatch } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback } from 'react';
+import { useAnalytics } from '../../app/analytics';
+import { useTaxName } from '../../app/hooks/use-country-list';
+import { useGeoLocationQuery } from '../../app/queries/geolocation';
+import InlineSupportLink from '../../components/inline-support-link';
 import { PageHeader } from '../../components/page-header';
 import PageLayout from '../../components/page-layout';
+import { localizeUrl } from './localize-url';
+import useRecordUserTaxEvents from './use-record-user-tax-events';
+import useUserTaxDetails from './use-user-tax-details';
+import UserTaxForm from './user-tax-form';
+import './style.scss';
+
+const HELP_CENTER_STORE = 'automattic/help-center';
+
+export default function UserTaxInfoPage() {
+	const { recordTracksEvent } = useAnalytics();
+	const translate = useTranslate();
+	const { data: geoData } = useGeoLocationQuery();
+	const { fetchError, userTaxDetails } = useUserTaxDetails();
+	const taxName = useTaxName(
+		userTaxDetails.country ?? userTaxDetails.country ?? geoData?.country_short ?? 'GB'
+	);
+	const resetSupportInteraction = useResetSupportInteraction();
+
+	const { setShowHelpCenter, setNavigateToRoute } = useDataStoreDispatch( HELP_CENTER_STORE );
+
+	const reduxDispatch = useDispatch();
+
+	/* This is a call to action for contacting support */
+	const contactSupportLinkTitle = translate( 'Contact Happiness Engineers' );
+
+	const taxSupportPageURL = localizeUrl( 'https://wordpress.com/support/vat-gst-other-taxes/' );
```

Comment:
This PR is bringing in _a lot_ of code for this. I'm not sure it is something we need to include in this PR. @arthur791004 you've set up linking to support pages elsewhere in the HD right? Have we got support link localisation working elsewhere yet?

In either case, I think this is something we should mark as todo and remove from this PR, because it is something we're going to need to solve everywhere, not just for the tax details.

Reply:
> In either case, I think this is something we should mark as todo and remove from this PR, because it is something we're going to need to solve everywhere, not just for the tax details.

Yes, agree!

Reply:
+1